### PR TITLE
Uncomment failing tests with few changes

### PR DESCRIPTION
**Pull request by tymefighter**

### CODE REVIEW

File: test/Streamly/Test/Internal/Data/Parser.hs [hunk 1]
```diff
@@ -450,6 +450,41 @@ sliceSepBy =
 --         Right _ -> False
 --         Left _ -> True)
 
+many :: Property
+many = 
+    forAll (listOf (chooseInt (0, 1))) $ \ls ->
+        let
+            concatFold = FL.Fold (\concatList curr_list -> return $ concatList ++ curr_list) (return []) return
+            prsr = P.many concatFold $ P.sliceSepBy (== 1) FL.toList
```
Comment (harendra-kumar) [inline test/Streamly/Test/Internal/Data/Parser.hs:458]: Can we have a separate test for `Control.Applicative.many` as well? Also, for `Control.Applicative.some`.

**Comment by pranaysashank** [comment]
Can you check why all the CIs are failing? Looks like a test failed testing the die combinator.

**Comment by harendra-kumar** [comment]
@tymefighter can you please address the review comments in the PR. Also, now fromParserK has been merged so all tests should pass.

**Comment by harendra-kumar** [comment]
Merged.

**Pull request closed without merging**